samples/cellular/udp/src/main.c

```diff
@@ -174,6 +189,21 @@ static int modem_connect(void)
 
 	return 0;
 }
+
+#if !defined(CONFIG_UDP_DATA_UPLOAD_FOREVER)
+static int modem_shutdown(void)
+{
+	int err;
+
+	err = nrf_modem_lib_shutdown();
+	if (err) {
+		printk("Failed to shutdown modem library, error: %d\n", err);
+		return err;
+	}
+
+	return 0;
+}
```

Comment:
There is no point to defined this function to call `nrf_modem_lib_shutdown()`; just call `nrf_modem_lib_shutdown()` directly.